go.sum

```diff
@@ -0,0 +1,107 @@
+github.com/BurntSushi/toml v0.3.1/go.mod h1:xHWCNGjB5oqiDr8zfno3MHue2Ht5sIBksp03qcyfWMU=
```

Comment:
What do you think about updating the README re: how to install dependencies. It would be helpful to update how to get the project up and running along the way.